ofl/dawningofanewday/DESCRIPTION.en_us.html

```diff
@@ -1 +1,6 @@
-<p>Dawning of a New Day is based on the handwriting of a friend.  The title was chosen by my friend, but also had great meaning to me. I like to think of each new day as a chance to start fresh, free from past mistakes. It is my desire that the light, fluid motions of this script mimic that hopeful feeling.</p>
\ No newline at end of file
+<p>
+Dawning of a New Day is based on the handwriting of a friend of the principal designer.
+The title was chosen by my friend, but also had great meaning to them.
+They like to think of each new day as a chance to start fresh, without past mistakes.
```

Comment:
Suggestion looking for pronouns consistency based on original version:

The title was chosen by the designer's friend but also had great meaning to the author as she likes to think of each new day as a chance to start fresh, without past mistakes.